### PR TITLE
Add development process steps

### PR DESCRIPTION
**Pull request by timgl**
On the fence whether this should live in handbook (given it's internal focus) or docs (since it does impact contributors)

### CODE REVIEW

**Comment by jamesefhawkins** [comment]
Docs are to do with code and running Posthog

This is a process and thus is a company thing so should be in handbook

You could link to it from contributing.md in the repo

**Comment by EDsCODE** [comment]
Echoing @jamesefhawkins. I've also seen these guides usually linked at the bottom of the main repo readme to somewhere on the website for more information (or handbook in these case).

File: docs/development-process/development-process.md [hunk 1]
```diff
@@ -1,6 +1,66 @@
...
+
+## 2. Sizing a task
+
+When picking up a task, it should be do-able in a day, including code review and QA. If it's not, you need to break it down into smaller chunks until it is. Tasks of this size are easy to test, easy to deploy, won't often cause merge conflicts and should still deliver some kind of value.
+
+Even if you're contributing, this is helpful as it means you'll be able to contribute to PostHog faster.
+
+## 3. Writing code
+
+We're big fans of TDD. We've tried to create test infrastructure that helps you rather than annoys you. If that isn't the case, please raise an issue! Keeping tests is a high priority to keep developer productivity high.
```
Comment (EDsCODE) [inline docs/development-process/development-process.md:27]: Mention adding a PR to the docs as needed too

**Comment by timgl** [comment]
I've put this in the handbook as per discussion

**Pull request closed without merging**